packages/client/package.json

```diff
@@ -70,10 +70,10 @@
     "sass": "1.50.0",
     "socket.io-client": "4.4.1",
     "ts-node": "10.8.2",
-    "vite": "2.9.4",
+    "vite": "3.1.0",
     "vite-plugin-compression": "0.5.1",
-    "vite-plugin-html": "2.1.2",
-    "vite-plugin-optimize-persist": "0.1.2",
+    "vite-plugin-html": "3.2.0",
+    "vite-plugin-optimize-persist": "^0.1.2",
```

Comment:
Is this carrot `^` required here? Because the latest version for this package is already `'0.1.2'`. And I think for most of the packages we are specifying the exact version instead of carrot based notations.

Reply:
ah yes, good catch, we can remove this dep entirely as its no longer used